src/util/util_errors.h

```diff
@@ -85,6 +85,7 @@ enum sssd_errors {
     ERR_INPUT_PARSE,
     ERR_NOT_FOUND,
     ERR_DOMAIN_NOT_FOUND,
+    ERR_NO_DOMAIN_ENABLED,
```

Comment:
I have just one minor concern here.
So far `ERR_INVALID_FILTER` was error _X_. Now error _X_ is `ERR_NO_DOMAIN_ENABLED` and `ERR_INVALID_FILTER` (and all the following errors) where increased by one.
Is it possible that these values arrive somehow to the user (for instance, `main()` returns these error codes)? In that case, this modification changes that tool's behavior because previously it would return _Y_ is now it will return _Y+1_ (if _Y_ >= _X_).
Is this a problem?

Reply:
Not a problem. We rather place it to a relevant position. sss_cache does not have any documented exit codes.